packages/client/hmi-client/src/components/resources/resources-list.vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+import { Project, ProjectAssetTypes } from '@/types/Project';
+import { RouteMetadata, RouteName } from '@/router/routes';
+import { Resource } from '@/types/Resource';
+import ResourceCard from '@/components/resources/ResourceCard.vue';
+import useResourcesStore from '@/stores/resources';
+import { RouteParamsRaw, useRouter } from 'vue-router';
+
+const props = defineProps<{
+	project: Project;
+	resourceRoute?: RouteName;
+	// maybe add list size later
+}>();
+
+const emit = defineEmits(['show-data-explorer']);
+const goToDataExplorer = () => emit('show-data-explorer');
+
+const router = useRouter();
+const activeProjectAssets = useResourcesStore().activeProjectAssets;
+
+const resources: Resource[] = [];
+
+const filteredRouteMetadata: {
```

Comment:
_Optional nitpick_:

I thought `filteredRouteMetadata.metadata` and `filteredRouteMeatdata.route` were a bit verbose. If you structured this object like 

```
const filteredRouteMetadata: {
	route: RouteName,
	displayName: string,
	icon: any,
	projectAsset?: ProjectAssetTypes
}[] = props.resourceRoute
		? [{ route: props.resourceRoute, ...RouteMetadata[props.resourceRoute] }]
		: Object.entries(RouteMetadata)
			.map(([route, metadata]) => [{ route: route as RouteName, displayName: metadata.displayName, icon: metadata.icon, projectAsset: metadata.projectAsset }])
			.flat();
```

then in your loop you can destructure `filteredRouteMetadata` like `const { route, displayName, icon, projectAsset } = filteredRoute;`

Which will be more terse overall!